docs/DEV_SETUP.md

```
@@ -0,0 +1,32 @@
+# Developer Setup (JDK 17, Gradle, Protobuf)
+
+This guide helps you build and run the Apache Wave server (and later the GWT client) on JDK 17.
+
+Prerequisites
+- Java: JDK 17 installed and discoverable by Gradle toolchains
+  - Suggested: sdkman
+    - curl -s "https://get.sdkman.io" | bash
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)